tests/unit/output/test_local.py

```diff
@@ -21,6 +23,35 @@ def test_save_missing(self):
                 o.save()
 
 
+@pytest.mark.usefixtures("dvc_repo")
```

Comment:
is this necessary?

Reply:
If yes, then simply using as argument will do the job.

Reply:
Looks like it's not necessary. Removing as `erepo` fixture already creates one.